styles/listenbrainz/catppuccin.user.css

```diff
@@ -0,0 +1,628 @@
+            <polygon class="c" points="14 1 26 8 26 22 14 29 14 1"/>
+          </svg>'
+          );
+          content: url("data:image/svg+xml,@{svg}");
+        }
+
+        #side-nav {
+          background-color: @mantle;
+          a,
+          a:visited {
+            color: @blue;
+            &:hover {
+              color: @sky;
+            }
+          }
+
+          .search-bar input {
+            background-color: @surface0;
+            color: @text;
+            &:focus {
+              box-shadow: none;
+            }
+          }
+
+          .mobile-nav-fix {
+            background-color: transparent;
+          }
+        }
+
+        #side-nav-overlay {
+          background-color: @crust;
+        }
+      }
+    }
+
+    a {
+      color: @blue;
+      &:hover {
+        color: @sky;
+      }
+      &:visited:not(.btn) {
+        color: @lavender;
+        &:hover {
+          color: @sky;
+        }
+      }
+    }
+
+    h2,
+    h3,
+    h4 {
+      color: @text;
+    }
+
+    .text-muted,
+    .help-block {
+      color: @subtext0;
+    }
+
+    .text-success {
+      color: @green;
+    }
+
+    a > svg.fa-link {
+      color: @text;
+    }
+
+    .form-control {
+      border-color: @base;
+      background-color: @surface0;
+      color: @text;
+    }
+
+    .panel {
+      background-color: @surface0;
+      color: @text;
+      border-color: @surface1;
+      .panel-heading {
+        background-color: @surface1;
+        outline-color: @surface2;
+      }
+    }
+
+    .well {
+      background-color: @surface0;
+      border-color: @mantle;
+    }
+
+    .toggle-switch {
+      background-color: @surface0;
+      &::before {
+        background-color: @overlay0;
+      }
+    }
+    .toggle-checkbox:checked + .toggle-switch {
+      background-color: @surface1;
+      &::before {
+        background-color: @accent-color;
+      }
+    }
+
+    .table {
+      tr {
+        > td,
+        > th {
+          border-color: @surface1;
+        }
+      }
+
+      &.table-striped > tbody > tr:nth-of-type(2n + 1) {
+        background-color: @surface0;
+      }
+    }
+
+    .music-service-selection .music-service-option {
+      label {
+        background-color: @surface1;
+        border-color: fade(@green, 50%);
+      }
+
+      input[type="radio"]:checked + label {
+        background-color: fade(@green, 50%);
+        &::after {
+          color: @green;
+          border-color: @green;
+          background-color: @surface1;
+        }
+      }
+    }
+
+    .pill.secondary {
+      color: @text;
+      border-color: @subtext0;
+    }
+
+    .btn-info,
+    .btn-outline,
+    .musicbrainz-profile-button,
+    .lb-follow-button.block,
+    .pill.secondary.active {
+      color: @text;
+      background-color: fade(@accent-color, 30%);
+      border-color: transparent;
+      &:hover {
+        color: @text;
+        background-color: fade(@accent-color, 40%);
+      }
```

Comment:
```suggestion
      color: @crust;
      background-color: @accent-color;
      border-color: transparent;

      &:hover {
        background-color: darken(@accent-color, 5%);
      }
```

Reply:
Hmm, this change looks a _lot_ nicer on dark flavours, which I like! However the button text becomes harder to read on Latte. Perhaps a separate style for Latte in particular might be needed. 🤔

Reply:
Huh really? How does it look on latte for you?

Reply:
Not *awful*, but definitely less legible than the previous version to my eye.
<img width="294" alt="Screenshot 2024-05-25 at 13 23 20" src="https://github.com/catppuccin/userstyles/assets/610876/a352e4a9-3721-419a-9c86-ac9fa7342bba">
<img width="302" alt="Screenshot 2024-05-25 at 13 23 54" src="https://github.com/catppuccin/userstyles/assets/610876/2bdbf231-8ca5-4f28-9c12-b907cd21e822">
I generally find light themes too bright anyway, though, so I may not be the best judge. 🤷‍♀️

Reply:
I tried slightly `lighten()`ing only the Latte version of crust, and I'm finding this a lot more legible. Otherwise basically as suggested. 👍
<img width="291" alt="Screenshot 2024-05-27 at 10 13 27" src="https://github.com/catppuccin/userstyles/assets/610876/297ca6b4-e145-45c7-b591-50abfa3a15b6">